main.py

```
@@ -0,0 +1,146 @@
+class EditProfileResponse(BaseModel):
+    status: str
+    data: EditProfileData
+
+@app.patch("/users/{nickname}", response_model=EditProfileResponse)
+async def update_profile(nickname: str):
+    return {
+        "status": "success",
+        "data": {
+            "nickname": nickname,
+            "profile_image": "profile_image",
+            "updated_at": datetime.now()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)